docs/HowToMonitorWithSumoLogic.md

```diff
@@ -0,0 +1,172 @@
+### NuoDB Insights and Sumo Logic
+
+---
+
+Sumo Logic is an observability suite offering end-to-end observability and security tools within one platform. Sumo Logic is an alternative monitoring solution to the current NuoDB Insights suite.
+
+### Sumo Logic Dashboards Setup
+
+---
+
+1. [Sumo Logic Collectors](#sumo-logic-collectors) setup for NuoDB Insight data.
+
+2. [NuoDB Helm Charts](#nuodb-helm-charts) modifications to enable data collection.
```

Comment:
Suggest to change `modifications` to `configuration`.

Reply:
Updated to "configuration"